core/views.py

```diff
@@ -63,17 +66,15 @@ def get(self, request, format=None):
         See the following example bellow:
         """
         # list of token roles
-        print(request.roles)
-        return super().get(self, request) 
+        return JsonResponse({"message": request.roles})
 
 
-@keycloak_roles({'POST': ['director', 'employee']})
-@api_view(['POST'])
-def refinance_loan(request):
+@keycloak_roles(['director', 'judge', 'employee'])
+@api_view(['GET'])
+def loans(request):
     """
-    Refinance loan endpoint
-    This endpoint has configured keycloak roles only POST method 
-    and only POST methods will be accepted in api_view.
+    List loan endpoint
+    This endpoint has configured keycloak roles only GET method 
+    and only GET methods will be accepted in api_view.
     """
-    print(request.roles)
-    return JsonResponse({"detail": "success"}, status=status.HTTP_200_OK)
\ No newline at end of file
+    return JsonResponse({"message": request.roles})
```

Comment:
did it work for you? how do the roles from the keycloak token get into request.roles here? in my project, when I updated the branch, I started getting a 403 error because request.roles always contains an empty list

Reply:
Yeap, I did. Beyond running unit tests, I've used Mock() function like this:

```python
KeycloakConnect.roles_from_token = Mock(return_value=['director'])
```
I've used it in both methods: `test_when_token_is_active_and_has_roles_request_not_authorizated` and `test_when_token_is_active_and_has_roles_request_authorizated`.